.github/workflows/sync-labels.yml

```diff
@@ -0,0 +1,28 @@
+name: Sync Labels
+
+on:
+  label:
+    types: [created, edited, deleted]
+  push:
+    branches:
+      - master
+
+jobs:
+  label-sync:
+    name: Install & Run
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_API_TOKEN: ${{ secrets.GITHUB_API_TOKEN }}
```

Comment:
This is using my personal access token. We'd probably want to give it a less permissive token from @hyphacoop-bot

Reply:
```suggestion
      # Personal access token for `hyphacoop-bot` with `repo` permission
      GITHUB_API_TOKEN: ${{ secrets.GITHUB_API_TOKEN }}
```